File: samples/nodejs/src/encryption/AuthenticatedCiphertext.ts
import {Ciphertext} from "./Ciphertext";
import {AuthenticationTag} from "./AuthenticationTag";

export class AuthenticatedCiphertext {
    public readonly ciphertext: Ciphertext
    public readonly tag: AuthenticationTag


    constructor(ciphertext: Ciphertext, tag: AuthenticationTag) {
        this.ciphertext = ciphertext;
        this.tag = tag;
    }
}

File: samples/java/persistence/src/main/java/io/sytac/azure/demos/persistence/encryption/AuthenticatedCiphertext.java
package io.sytac.azure.demos.persistence.encryption;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

@Data @Builder
@AllArgsConstructor
public class AuthenticatedCiphertext {
    private Ciphertext ciphertext;
    private AuthenticationTag tag;
}
